### PR TITLE
adding keyboard

### PR DESCRIPTION
**Pull request by SonOfAres**
## Description

Adding a custom made PCB to QMK. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (**addition** or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/iriskeyboards/readme.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# iriskeyboards
+
+![IrisRev0](https://i.imgur.com/sCrZ43r.jpg)
```
Comment (fauxpark) [inline keyboards/iriskeyboards/readme.md:3]:
```suggestion
![IrisRev0](https://i.imgur.com/sCrZ43rh.jpg)
```

File: keyboards/iriskeyboards/iriskeyboards.json [hunk 2]
```diff
@@ -0,0 +1,239 @@
+{
```
Comment (fauxpark) [inline keyboards/iriskeyboards/iriskeyboards.json:1]: This file goes in the VIA repo, not here.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/iriskeyboards/iriskeyboards.json (hunk 2 above)
Comment (SonOfAres) [reply keyboards/iriskeyboards/iriskeyboards.json:1]: Sorry about that! 

Thank you for what you guys do, QMK is fantastic!

**Review by SonOfAres — COMMENTED** [review]
I have done the appropriate changes on the requested changes.

File: keyboards/iriskeyboards/info.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (fauxpark) [inline keyboards/iriskeyboards/info.json:1]: This needs to be filled out. Press Ctrl+Shift+I in the Configurator to preview it.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/iriskeyboards/info.json (hunk 3 above)
Comment (SonOfAres) [reply keyboards/iriskeyboards/info.json:1]: I have filled it out and tested it using Ctrl+Shift+I in the Configurator!

**Review by SonOfAres — COMMENTED** [review]
updated website URL

File: keyboards/iriskeyboards/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    ),
+
+    [3] = LAYOUT_all(
+        KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS,  KC_TRNS,                            KC_TRNS,                            KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+
```
Comment (fauxpark) [inline keyboards/iriskeyboards/keymaps/default/keymap.c:64]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by ridingqwerty — APPROVED** [review]
Looks good to me.

**Pull request merged**